### ISSUE TITLE
Performance improvement of multi-row inserts with the bypass of binding values

### ISSUE THREAD

**Issue by deepaksenthilkumar** [dbeaver/dbeaver#13498]
**Is your feature request related to a problem? Please describe.**
Performance issues in multi-row inserts of millions of records for the Redshift database. The issue is because of the time the Redshift 1.x JDBC driver takes to prepare a statement (which is exponentially proportional to the number of question marks in the statement we are trying to prepare). 

**Describe the solution you'd like**
Provide a new option in Data Transfer Wizard (so the user can toggle it) to bypass binding values. You can pass this option in prepareQueryForStatement and if it is set, then pass values directly (or at least use SQLDialect#escapeScriptValue to somehow escape values) without binding them. And this option must be exposed in the UI with the descriptive tip with its cons and pros.

**Describe alternatives you've considered**
The single row insert is slow too, so we do need to implement this solution.

**Additional context**
N/A

**Comment by Matvey16** [comment]
@deepaksenthilkumar Something went wrong because with this option toggled on, only one row is inserted millions of times
![6B53Kp6xbx](https://user-images.githubusercontent.com/82543000/130256525-7468521e-82d0-4196-8285-599e3fb83972.gif)

**Comment by serge-rider** [comment]
Fixed.
Note: multi-row mode and value binding are independent parameters now.

Binding disable + big multi-row value increases data transfer performance dramatically.

**Comment by deepaksenthilkumar** [comment]
I noticed the bug. Thanks @serge-rider for fixing it, but using replace of ? is 6x slower during the replace. I will work on appending it to stringbuilder directly.

**Comment by deepaksenthilkumar** [comment]
Submitted the PR for a faster/accurate version of skip bind values. Thanks @Matvey16  for finding the issue!

**Comment by serge-rider** [comment]
Umm, from my expirience 99% of time it spends in insert itself and in network operations. Optimizing StringBuilder operations doesn't make much sense.

My concern is that we have a lot of different modes (batch, single/multi row, disabled bindings, databases which don't support batches, etc). All these cases must be tested.

**Comment by deepaksenthilkumar** [comment]
In this case, the slowness is because of how the replace method internally works vs. just append. I measured timing and it was just doing replace for 50-70 seconds for every run (100k multirow insert size)

I had the same concern and tested all the below options as I didn't want to break anything.

These are the options that I tested.
1. Single record (multirow insert+no skip bind value, disable batches+no skip bind value, multirow insert+skip bind value, disable batches+skip bind value)
2. 2k+ records (multirow insert+no skip bind value, disable batches+no skip bind value, multirow insert+skip bind value, disable batches+skip bind value)
3. 1M+ records (multirow insert+skip bind value) 

The modified version, functionally does the same thing as what and @LonwoLonwo  had for skip bind values and multirow respectively. It just binds the variable in a loop vs. doing replace later.
